CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/FlowNetwork.java

```diff
@@ -32,16 +32,16 @@
 
     private static final Logger LOGGER = Logger.getLogger(FlowNetwork.class.getName());
 
-    private int[] nodeOutDegree;
-    private double[] sumLinkOutWeight; // Per leaf nodes
-
     private double[] nodeFlow;
     private double[] nodeTeleportRates;
     private Connection[] flowConns;
 
     public void calculateFlow(final Network network, final Config config) {
         LOGGER.log(Level.INFO, "Calculating global flow... ");
 
+        int[] nodeOutDegree;
+        double[] sumLinkOutWeight; // Per leaf nodes
```

Comment:
can these be defined at the point they're instantiated (possibly made final if it makes sense)?